.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 doctests = True
-exclude = .venv,.git,.tox,dist,doc,*lib/python*,*egg,build,*web-server/*,
+exclude = .venv,venv,.git,.tox,dist,doc,*lib/python*,*egg,build,*web-server/*,
```

Comment:
Can we pull this out into another PR?

Reply:
Certainly....

Reply:
Instead of pulling this change to a new PR, I rejiggered my work environment to use `.venv` instead of `venv`.